src/index.js

```diff
@@ -57,8 +58,20 @@ async function main() {
       core.setFailed(`Assertions have failed.`)
       // continue
     }
+
+    const slackWebhookUrl = input.slackWebhookUrl
+
+    if (slackWebhookUrl) {
```

Comment:
Should it send a message on error or for every run?
My solution: only on error, but there should be an option to do it for every run.

Reply:
Might be a good idea to introduce a noise/log level flag at this point? Like `info|error`, `info` outputs on every run to the desired output method(s), and `error` only outputs on assertion failures/LH errors?

Reply:
I like idea with `info|error`, cc @alekseykulikov